.github/codeql/codeql-config.yml

```diff
@@ -0,0 +1,4 @@
+paths-ignore:
+  - ./data
```

Comment:
We do not have any `data` directory checked-in in the repository. If unnecessary, please remove it?
```suggestion
```